.github/workflows/ipa-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     container: registry.fedoraproject.org/fedora:${{ matrix.os }}
     strategy:
       matrix:
-        os: ['32', '33']
+        os: ['32', '33', '34']
```

Comment:
This change will enable the build job on F34, but the tests aren't enabled yet for F34. See lines 65 and 253.

Reply:
Thanks edewata. I could have sworn I actually saw the f34 tests running when watching the progress. I must have just wished it :) Will have a look.